src/app/qrcode/qr-wifi/page.tsx

```diff
@@ -0,0 +1,390 @@
+              <p className='inline text-[16px] font-[500] md:text-[20px]'>
+                Encryption
+              </p>
+              <Input
+                fontSize={inputFontSize}
+                textValue={inputEncryption}
+                dropdownOptions={['WPA/WPA2', 'WEP', 'NONE', 'RAW']}
+                onDropdownSelect={(selectedOption: any) => {
+                  setInputEncryption(selectedOption);
+                }}
+                collapseIcon={true}
+                className='ms-[24px] w-[132px] md:ms-2'
+                height={inputHeight}
+              />
+            </div>
+            {isLoaded && isLoggedIn && (
+              <div className=' mb-4 ml-8 flex items-center justify-between md:mb-[8px] md:me-[20px] md:inline-flex'>
+                <p className='inline text-[12px] font-[500] text-black md:text-[16px]'>
+                  Organization
+                </p>
+                <Input
+                  collapseIcon
+                  height={inputHeight}
+                  className='ms-[4px] w-[200px] md:ms-[8px]'
+                  fontSize={inputFontSize}
+                  textValue={organizationValue!.shortName}
+                  dropdownOptions={organizationOptions!}
+                  onDropdownSelect={
+                    handleChange(ShortenInputFieldEnum.ORGANIZATION) as (
+                      value: ShortenInputFieldType,
+                    ) => void
+                  }
+                />
+              </div>
+            )}
+          </div>
+        </div>
+        <div className='mb-[16px] md:flex md:items-center md:justify-between'>
+          <h6 className='mb-[4px] text-[16px] font-[500] md:mb-0 md:inline md:text-[20px]'>
+            Password
+          </h6>
+          <div className='relative flex items-center md:ml-[20px] md:flex-grow'>
+            <Input
+              fontSize={inputFontSize}
+              type={passwordVisible ? 'text' : 'password'}
+              iconSrc='/icons/qrcode/password.svg'
+              iconAlt='password-icon'
+              placeholder='Enter your password'
+              textValue={inputPassword}
+              divider={true}
+              onInput={setInputPassword}
+              height={inputHeight}
+              onEnter={() => {
+                logWifiInfo();
+              }}
+              className='pr-8'
+            />
+            <button
+              className='absolute right-2 inline-block rounded-full hover:bg-gray-300'
+              onClick={() => setPasswordVisible(!passwordVisible)}
+            >
+              <Image
+                src={getIcon(
+                  '/icons/qrcode',
+                  'password_visible.svg',
+                  !passwordVisible ? Icon.ACTIVE : Icon.INACTIVE,
+                )}
+                alt='password-visible-icon'
+                width={20}
+                height={20}
+              />
+            </button>
```

Comment:
```suggestion
            <Input
              fontSize={inputFontSize}
              type={passwordVisible ? 'text' : 'password'}
              iconSrc='/icons/qrcode/password.svg'
              iconAlt='password-icon'
              placeholder='Enter your password'
              textValue={inputPassword}
              divider={true}
              onInput={setInputPassword}
              height={inputHeight}
              onEnter={() => {
                logWifiInfo();
              }}
              className='pr-8'
            />
```
The Input component is supposed to support both left and right icon.
-> Remove the button for show/hide password, accept this as a bug that will be fixed in later PRs (by modifying the Input component to accept multiple icons)